### ISSUE TITLE
new quickstarter for Lua API Gateway and WAF implementations based on openresty-nginx shared-image

### ISSUE THREAD

**Issue by gerardcl** [opendevstack/ods-core#67]
This new quickstarter will provide fast API Gateway implementations based on OpenResty and Lua scripting.

Moreover, it will give the chance to provide/enable WAF support based on ModSecurity Nginx module.

Proposed name: be-openresty-waf

**Comment by clemensutschig** [comment]
@gerardcl  - this should move over to ods-project-quickstarters, no?

**Comment by gerardcl** [comment]
Duplicate of #

**Comment by gerardcl** [comment]
right!
at least for this first release without a testing slave image for lua, so what I will do is to keep it here but change a little bit the naming and related to the quickstarters

**Comment by clemensutschig** [comment]
@gerardcl  - why is this in -core? the slaves & boilerplates are in -quickstarter ... ?

**Comment by gerardcl** [comment]
You are right! this is legacy from first iterations related to creating its base image in shared-images. I'll move it ASAP.
